H.Ws/H.W_3_no pre-commit.py

```diff
@@ -0,0 +1,89 @@
+from pyspark import StorageLevel
+from pyspark.sql import SparkSession
+
+appName = "PySpark Example - MariaDB Example"
+master = "local"
+
+# Create Spark session
+spark = SparkSession.builder.appName(appName).master(master).getOrCreate()
+
+sql = "select Hit, atBat, batter, game_id from baseball.batter_counts"
+database = "baseball"
+user = "root"
+password = "root"
+server = "localhost"
+port = 3306
+jdbc_url = f"jdbc:mysql://{server}:{port}/{database}?permitMysqlScheme"
```

Comment:
I like your use of permit mysql scheme. I did not use this in my jdbc_url and had to include a zeroifnull for the dates that were 0. This might be an alternate solution to that?

Reply:
I think so, cause I didn't have this issue